### ISSUE TITLE
Android 27 broken now

### ISSUE THREAD

**Issue by bannsec** [frida/frida#1046]
I've been able to previously use Frida with the Android 27 image. However, now when I attempt to connect to the device, i get:

```
NotSupportedError: unable to inject library into process without libc
```

This is a base android x86_64 image API 27 which worked before Android 10 support was introduced.

**Comment by bannsec** [comment]
For reference, the last working version for Android 27 appears to be `12.7.7`.

**Comment by bannsec** [comment]
Looks like this is fixed now in 12.7.10. Closing issue.

**Comment by oleavr** [comment]
Thanks for reporting! Another related regression fix is being released in 12.7.11 right now.
